types/weixin-app/index.d.ts

```diff
@@ -3283,22 +3304,9 @@ declare namespace wx {
 	 *
 	 */
 	function openCard(options: OpenCardOptions): void;
-	// 开放接口-----设置
-	interface AuthSetting {
-		scope:
-			| "scope.userInfo"
-			| "scope.userLocation"
-			| "scope.address"
-			| "scope.invoiceTitle"
-			| "scope.werun"
-			| "scope.record"
-			| "scope.writePhotosAlbum";
-		success?(res: ErrMsgResponse): void;
```

Comment:
Are you sure callback function is not needed here?

Reply:
The type of `res` inside `OpenSettingOptions`'s `success` callback was wrong.

The wrong usage before this PR merged:

```ts
wx.openSetting({
  success(res) {
    // res.authSetting is AuthSetting
    // res.authSetting.success is function
  }
});
```

The correct usage:

```ts
wx.openSetting({
  success(res) {
    // res.authSetting is Scope
  }
});
```

Reply:
Right.

Reply:
Thanks. Could you please approve the PR if every thing OK ?